src/navigation/HomeStackNavigator.tsx

```diff
@@ -0,0 +1,87 @@
+import { createStackNavigator } from '@react-navigation/stack'
+import React from 'react'
+import { widthPercentageToDP as wp } from 'react-native-responsive-screen'
+import { useTheme } from 'styled-components'
+
+import labels from '../constants/labels.json'
+import useReadSelectedProfessions from '../hooks/useReadSelectedProfessions'
+import { useTabletHeaderHeight } from '../hooks/useTabletHeaderHeight'
+import DisciplineSelectionScreen from '../routes/DisciplineSelectionScreen'
+import ExercisesScreen from '../routes/ExercisesScreen'
+import ImprintScreen from '../routes/ImprintScreen'
+import ProfessionSelectionScreen from '../routes/ProfessionSelectionScreen'
+import ResultDetailScreen from '../routes/ResultDetailScreen'
+import ResultScreen from '../routes/ResultScreen'
+import AddCustomDisciplineScreen from '../routes/add-custom-discipline/AddCustomDisciplineScreen'
+import HomeScreen from '../routes/home/HomeScreen'
+import ManageSelectionsScreen from '../routes/manage-selections/ManageSelectionsScreen'
+import ScopeSelection from '../routes/scope-selection/ScopeSelectionScreen'
+import screenOptions from '../services/screenOptions'
+import { RoutesParams } from './NavigationTypes'
+
+const Stack = createStackNavigator<RoutesParams>()
+
+const HomeStackNavigator = (): JSX.Element | null => {
+  const { data: professions, loading } = useReadSelectedProfessions()
+
+  const headerHeight = useTabletHeaderHeight(wp('15%'))
+  const options = screenOptions(headerHeight)
+  const { manageDisciplines, overview } = labels.general.header
+  const theme = useTheme()
+
+  if (loading) {
+    return null
+  }
+
+  return (
+    <Stack.Navigator
+      initialRouteName={professions !== null ? 'Home' : 'ScopeSelection'}
```

Comment:
(optional as i guess this is no change from you): Avoid using negative comparision.